CHANGELOG.md

```diff
@@ -39,24 +40,55 @@ These changes likely break some apps, please read the
   - `widget.SplitContainer` (now `container.Spilt`)
   - `widget.Group` (replaced by `widget.Card`)
   - `widget.Box` (now `container.NewH/VBox`)
+* Many deprecated fields have been removed, replacements listed in API docs 1.4
+  - for specific information you can browse https://developer.fyne.io/api/v1.4/
 
 ### Added
 
-* Add MouseButtonTertiary for middle mouse button events on desktop
+* Data binding API to connect data sources to widgets and sync data
+  - Add preferences data binding and `Preferences.AddChangeListener`
+  - Add bind support to `Check`, `Entry`, `Label`, `List`, `ProgressBar` and `Slider` widgets
+* Animation API for handling smooth element transitions
+  - Add animations to buttons, tabs and entry cursor
+* Storage repository API for connecting custom file sources
+  - Add storage functions `Copy`, `Delete` and `Move` for `URI`
+  - Add `CanRead`, `CanWrite` and `CanList` to storage APIs
+* New Theme API for easier customisation of apps
+  - Add ability for custom themes to support light/dark preference
+  - Support for custom icons in theme definition
+  - New `theme.FromLegacy` helper to use old theme API definitions
 * Add fyne.Vector for simple x/y coordinates
```

Comment:
Does `fyne.Vector` use `int` or `float32`?

It should be noted here, simply because other changes note this (Pos, TextSize, etc.)